programmers/2일차/수박수박수박수.js

```diff
@@ -0,0 +1,23 @@
+function solution (n) {
+    const word  = "수박";
+    const split_word = word.split("");
+    const answer =[];
+    let count = 0;
+    while(n>count) {
```

Comment:
```suggestion
    let count = 0;
    
    while (n > count) {
```
위에 말씀드린 것 처럼  변수 선언 이후에 줄 바꿈 및   while 공백  (조건식) 공백 {  식으로 작성해주셔야 해요